README.md

```diff
@@ -182,6 +182,13 @@ There are a few important features and restrictions in this destination:
 
 This destination is using [Instabug iOS SDK](https://github.com/Instabug/Instabug-iOS) for logging implementation.
 
+### Analytics
+
+Analytics tracking is optional in logger framework, if client want to include analytics tracking they need to implement `AnalyticsProviderProtocol` and provide information through protocol and import `OktaLogger/Analytics` pod.
+Right now `Okta Logger` support tracking for `AppCenter` and client need to register key secret to Appcenter with in the application.
```

Comment:
We need to be more explicit here. We need to mention which exact pod has that implementation and what capabilities are